### PR TITLE
Adjust vsphere connected jobs

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-connected

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-vsphere-disconnecting

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-vsphere-upi-disconnecting-network-p2-f14

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-remote-worker-f14

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-vsphere-upi-disconnecting-network

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-remote-worker

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-connected

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-vsphere-upi-disconnecting-network

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-vsphere-upi-disconnecting-network-p2-f14

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-connected

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-vsphere-upi-disconnecting-network-p2-f14

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-vsphere-upi-disconnecting-network-p2-f14

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-connected

**Comment by sgaoshang** [comment]
@jianlinliu @jinyunma Could you help to review it? Thanks.

File: ci-operator/step-registry/vsphere/disconnecting-network/recover/vsphere-disconnecting-network-recover-commands.sh [hunk 1]
```diff
@@ -49,8 +49,8 @@ function check_latest_machineconfig_applied() {
 
 function wait_machineconfig_applied() {
     local role="${1}" try=0 interval=60
-    num=$(oc get node --no-headers | awk -v var="${role}" '$3 == var' | wc -l)
-    local max_retries; max_retries=$(expr $num \* 10)
+    num=$(oc get node --no-headers | grep -c "${role}")
```
Comment (jianlinliu) [inline ci-operator/step-registry/vsphere/disconnecting-network/recover/vsphere-disconnecting-network-recover-commands.sh:52]: How about this:
```
oc get node --no-headers -l node-role.kubernetes.io/"$role"= | wc -l
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 2]
```diff
@@ -1651,6 +1651,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
+- as: vsphere-upi-disconnecting-network-p2-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:1654]: we never added this ci job before?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 3]
```diff
@@ -3017,6 +3017,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
+- as: vsphere-upi-disconnecting-network-p2-f14
+  cron: 17 7 11,26 * *
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:3021]: How about f28?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml (hunk 2 above)
Comment (sgaoshang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:1654]: I think so

File: ci-operator/step-registry/vsphere/disconnecting-network/vsphere-disconnecting-network-commands.sh [hunk 4]
```diff
@@ -56,8 +56,8 @@ function check_latest_machineconfig_applied() {
 
 function wait_machineconfig_applied() {
     local role="${1}" try=0 interval=60
-    num=$(oc get node --no-headers | awk -v var="${role}" '$3 == var' | wc -l)
-    local max_retries; max_retries=$(expr $num \* 10)
+    num=$(oc get node --no-headers | grep -c "${role}")
```
[the remark this answers is not in the document]
Comment (sgaoshang) [reply ci-operator/step-registry/vsphere/disconnecting-network/vsphere-disconnecting-network-commands.sh]: updated in code.

**Comment by jianlinliu** [comment]
leave this to @jinyunma for a double confirm.

**Comment by sgaoshang** [comment]
@kuiwang02 hello, could you help to approve it? Thanks.

**Comment by kuiwang02** [comment]
@sgaoshang I am ok from golang's view.

please contact with qe-productivity to approve it. cc @jhou1 @liangxia

**Pull request merged**